test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md

```diff
@@ -12,6 +12,7 @@ products:
       - 1.4.0
       - 1.7.0
       - 1.10.0
+      - 1.13.0
 estimate: 1h
 tags:
   - destructive
```

Comment:
Maybe need add more details about credentials, how to connect (for new user). I did following, not sure that it's correct,  I didn't find tables. DB seems empty.
```
sh-4.2$ psql --host=rhsso-postgres-rhoam.redhat-rhoam-operator.svc.cluster.local --port=5432 --username=postgres --password --dbname=postgres
Password for user postgres: 
psql: FATAL:  password authentication failed for user "postgres"
sh-4.2$ psql --host=throw-away-postgres.redhat-rhoam-operator.svc.cluster.local --port=5432 --username=postgresuser --password --dbname=throw-away-postgres
Password for user postgresuser: 
psql (10.17)
Type "help" for help.

throw-away-postgres=# \dt
Did not find any relations.
throw-away-postgres=# select * from client;
ERROR:  relation "client" does not exist
LINE 1: select * from client;
```